packages/adapters/ethers/src/client.ts

```diff
@@ -1112,7 +1109,14 @@ export class EthersAdapter {
 
   private async syncAuthConnector(projectId: string, bypassWindowCheck = false) {
     if (bypassWindowCheck || typeof window !== 'undefined') {
-      this.authProvider = W3mFrameProviderSingleton.getInstance(projectId)
+      this.authProvider = W3mFrameProviderSingleton.getInstance({
+        projectId,
+        onTimeout: () => {
+          AlertController.open(ErrorUtil.ALERT_ERRORS.INVALID_APP_CONFIGURATION, 'error')
+          // eslint-disable-next-line no-console
+          console.error(ErrorUtil.ALERT_ERRORS.originNotWhitelistedSocials(window.origin))
```

Comment:
Why are the console and Alert component messages different? Can they not be the same and could we instead use `console.error` inside of the open function?